### PR TITLE
Patch fma biasgelubwd cpp test

### PR DESCRIPTION
**Pull request by jjsjann123**
1. added util struct `ThreadLocalFmaDisableOverwrite` to disable fma for local thread;
2. disable fma on `FusionBiasGeluBwd`

### CODE REVIEW

**Comment by jjsjann123** [comment]
!build

File: csrc/utils.h [hunk 1]
```diff
@@ -109,6 +109,16 @@ enum class DisableOption {
...
 };
 
+// used only for testing/debugging
+class TORCH_CUDA_CU_API ThreadLocalFmaDisableOverwrite {
+ public:
+  ThreadLocalFmaDisableOverwrite(bool flag = true);
+  ~ThreadLocalFmaDisableOverwrite();
+
+ private:
+  bool old_flag;
```
Comment (naoyam) [inline csrc/utils.h:119]: nit: 
```suggestion
  bool old_flag_;
```

**Review by naoyam — APPROVED** [review]
LGTM

**Comment by jjsjann123** [comment]
!build

**Pull request merged**